Gateway/Http/Client/TransactionAuthorization.php

```diff
@@ -69,6 +69,13 @@ public function __construct(
         $client->setUsername($webserviceUsername);
         $client->setPassword($webservicePassword);
 
+		$client->setAdyenPaymentSource($this->_adyenHelper->getModuleName(), $this->_adyenHelper->getModuleVersion());
+
+		$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+		$productMetadata = $objectManager->get('Magento\Framework\App\ProductMetadataInterface');
+
```

Comment:
Let's not use objectManager this is a old deprecated way. Best to use dependency injection in the constructor.